tests/test.py

```diff
@@ -9714,6 +9714,42 @@ def testClientIDRead(self):
         self.assertEqual(ex.exception.errno, MYSQL_ERR_QUERY_INTERRUPTED_CODE)
 
 
+class TestMySQLTextCharsetLiterals(TestMySQLTextTypeAwareDecryptionWithoutDefaults):
+    def testClientIDRead(self):
+        """
+        We don't directly support charset introducers[1], but at least make
+        sure it can handle _binary.
+
+        [1]: https://dev.mysql.com/doc/refman/8.0/en/charset-introducer.html
+        """
+
+        self.schema_table.create(bind=self.engine_raw, checkfirst=True)
+        id = get_random_id()
+        # DON'T EVER, EVER DO THIS
+        # Do not use direct string interpolation for sql values
+        # This is just a test example
```

Comment:
we can avoid using raw sql and build query with sqlalchemy:
```
>>> from sqlalchemy.sql import expression
>>> from sqlalchemy.sql import operators
>>> unary = expression.UnaryExpression(
      sa.Column("column"), 
      operator=operators.custom_op("_binary"), 
      type_=sa.Numeric)
>>> str(unary)
'_binary "column"'
```
Use `unary` instead of `self.test_table.c.value_bytes`